src/components/Navbar/index.tsx

```diff
@@ -56,16 +59,30 @@ export class NavbarBase extends React.Component<Props> {
   }
 
   render() {
-    const { user, currentVersion } = this.props;
+    const { currentVersion, reviewersHost, user } = this.props;
 
     return (
       <Navbar className={styles.Navbar} expand="lg" variant="dark">
         <Navbar.Brand className={styles.brand}>
           <div className={styles.info}>
             {currentVersion && (
-              <div className={styles.infoItem}>
-                {getLocalizedString(currentVersion.addon.name)}
-              </div>
+              <>
+                <div
+                  className={makeClassName(styles.infoItem, styles.addonName)}
+                >
+                  {getLocalizedString(currentVersion.addon.name)}
+                </div>
+                <div className={styles.infoItem}>
+                  <a
+                    className={styles.reviewerToolsLink}
+                    href={`${reviewersHost}/reviewers/review/${currentVersion.addon.slug}`}
+                    rel="noopener noreferrer"
+                    target="_blank"
+                  >
+                    {gettext('Back to reviewer tools')}
```

Comment:
You could add an icon in there to make the shortened *Reviewer Tools* work better:

```diff
diff --git a/src/components/Navbar/index.tsx b/src/components/Navbar/index.tsx
index d676be6..55a440a 100644
--- a/src/components/Navbar/index.tsx
+++ b/src/components/Navbar/index.tsx
@@ -2,6 +2,7 @@ import makeClassName from 'classnames';
 import * as React from 'react';
 import { Button, Navbar } from 'react-bootstrap';
 import { connect } from 'react-redux';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 import { gettext, getLocalizedString } from '../../utils';
 import CommentSummaryButton from '../CommentSummaryButton';
@@ -79,6 +80,7 @@ export class NavbarBase extends React.Component<Props> {
                     rel="noopener noreferrer"
                     target="_blank"
                   >
+                    <FontAwesomeIcon icon={['fas', 'arrow-left']} />
                     {gettext('Back to reviewer tools')}
                   </a>
                 </div>
```